05_deploy_bootstrap_vm.sh

```diff
@@ -143,4 +143,27 @@ ssh -o "StrictHostKeyChecking=no" core@$IP sudo podman run \
 # Create a master_nodes.json file
 jq '.nodes[0:3] | {nodes: .}' "${WORKING_DIR}/ironic_nodes.json" | tee ocp/master_nodes.json
 
+MASTER_INTERFACE="eth1"
+
+# Fix etcd discovery on bootstrap
+rm -rf ocp/machineconfigs
+mkdir -p ocp/machineconfigs/temp
+# Find master machine config name
+while [ -z $(ssh -o StrictHostKeyChecking=no "core@$IP" sudo ls /etc/mcs/bootstrap/machine-configs/master*) ]; do sleep 5; done
```

Comment:
would it be a potentially infinite loop? would it make sense to terminate it eventually?

Reply:
Yes, it would finish as soon as MCS assets are generated. If there is some error with generating those masters won't be able to fetch ignition.

It usually takes ~20-30 secs to complete